tests/runner/src/env.rs

```diff
@@ -412,4 +419,4 @@ impl Display for ResultDisplayer {
             }
         }
     }
-}
+}
```

Comment:
Please add a new line at the end of file

Reply:
ok,thx!